src/boss/boss.go

```diff
@@ -75,120 +75,36 @@ func (b *Boss) ScalingWorker(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if worker_count > 3 {
-		worker_count = 3
-		// TODO: config cap
+	if worker_count > Conf.Worker_Cap {
+		worker_count = Conf.Worker_Cap
 		log.Printf("capping workers at %d to avoid big bills during debugging\n", worker_count)
```

Comment:
better to send back an error message over HTTP so it doesn't get buried/missed in the log output